pkg/services/internal/whatsonchain/is_valid_root_test.go

```diff
@@ -83,12 +70,12 @@ func TestIsValidRootForHeight(t *testing.T) {
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
 			// given:
-			fix := tst.Given(t)
+			given := tst.Given(t)
 			if tc.setup != nil {
```

Comment:
this if is not needed, and I would rather assume that tc.setup must be always available - it can simply be empty function